taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Car, Driver
+
+
+def validate_license_number_field(value):
+    if not (len(value) == 8 and value[:3].isalpha()
+            and value[:3].isupper() and value[3:].isdigit()):
+        raise ValidationError(
+            "Field must be 8 characters long, first 3 characters must be "
+            "capital letters, last 5 characters must be numbers!"
+        )
```

Comment:
Handle each error separately, so that user will know what exactly is wrong with the license number